Filter/PagerfantaFilter.php

```diff
@@ -3,37 +3,37 @@
 namespace LAG\AdminBundle\Filter;
 
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\ParameterBag;
 
-class PagerfantaFilter
+class PagerfantaFilter extends RequestFilter
```

Comment:
The adapter is here to provide an interface between the data provider and the pager. It does not hold the request